certs.sh

```
@@ -7,12 +7,66 @@ ROOT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+    if [ -z "$value" ]; then
+        echo "$var_name is required. Aborted."
+        exit 1
+    fi
+
+    printf -v "$var_name" '%s' "$value"
+}
+
+if [ -f "./environment/shared.env" ]; then
+    set -a
+    . "./environment/shared.env"
```

Shellcheck (0.11.0)

[info] Not following: ./environment/shared.env was not specified as input (see shellcheck -x).

(SC1091)